test/unit/query/test-error-event.js

```diff
@@ -7,11 +7,13 @@ var server = common.createFakeServer();
 server.listen(common.fakeServerPort, function (err) {
   assert.ifError(err);
 
-  var query = connection.query('INVALID SQL');
+  var sql = 'INVALID SQL';
```

Comment:
If you can un-DRY the test and just put the literal SQL below as well, that would be good :) I don't like tests that are too DRY because it cases accidents.